deploy-service/common/src/main/java/com/pinterest/deployservice/scm/GithubManager.java

```diff
@@ -191,6 +191,7 @@ public Queue<CommitBean> getCommits(String repo, String startSha, boolean keepHe
         // TODO: Do not RETRY since it will timeout the thrift caller, need to revisit
         Map<String, String> params = new HashMap<String, String>();
         params.put("sha", startSha);
+        params.put("path", path);
```

Comment:
Is this a file path? will it change the existing behavior?

Reply:
it is a folder/file path.
it will not change the existing behavior.
path is in the api, but it doesn’t take effect when querying github.
now, it will take effect, if no path provided, it will consider the whole repo.